test/build_widget.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+Widget buildWidget(Widget child) {
+  return MaterialApp(
+    debugShowCheckedModeBanner: false,
+    title: 'Flutter Demo',
+    theme: ThemeData(
+      primarySwatch: Colors.blue,
+    ),
+    home: child,
+  );
+}
```

Comment:
add an extra line at the end, Ctrl + Alt + L should fix it

Reply:
done